### PR TITLE
Wip sgh simplifying edit

### PR DESCRIPTION
**Pull request by sgharms**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,224 +1,147 @@
...
+## Problem Statement
 
-Almost all users of the modern web expect a high level of responsiveness and a
-_seamless_ experience when visiting a website. There is research that shows that
-40 percent of visitors to a website will leave if the site takes more than 3
-seconds to load. This number becomes higher when mobile visitors are taken into
-account.
+When it comes to making engaging web sites, we often find ourselves needing to
+send a lot of data (text, images, media, etc.). But sending and loading this
+**huge** data payload when your first land on a page **feels** slow,
```
Comment (cjbrock) [inline README.md]: The word payload feels above our 6-8 grade target. Maybe go with something like "sending and loading this huge amount of data"?

Comment (cjbrock) [inline README.md]: Not great to repeat the same word twice in the same sentence, maybe use something descriptive like "feels like it can take forever" or "feels like it can take a long time" (without the hyperbole"?

File: README.md [hunk 2]
```diff
@@ -1,224 +1,147 @@
...
+especially on a slow connection.
 
-As web developers, making sure the websites we build are fast and responsive to
-users is critical. But when it comes to loading content, we often find ourselves
-needing a lot of data.
+However, web users expect sites to load quickly and to stay updated. Research
+shows that 40 percent of visitors to a website will leave if the site takes
+more than 3 seconds to load. Mobile users are even less patient.
```
Comment (cjbrock) [inline README.md]: I like adding the actual numbers here. The ambiguity was bugging me.

File: README.md [hunk 3]
```diff
@@ -1,224 +1,147 @@
...
 
-Data takes memory to store, and bandwidth to download from a server. But just as
-importantly to the end user, loading a lot of data at once can also take a
-noticeable amount time, especially on a slow connection.
+We deliver sites that don't bore users by using a technique called ***AJAX***.
```
Comment (cjbrock) [inline README.md]: "Don't bore users" seems like really negative phrasing to me. Maybe say "We deliver sites that stimulate users"?

File: README.md [hunk 4]
```diff
@@ -1,224 +1,147 @@
...
-
-Overall, allowing JavaScript to request, receive, and render data results in a
-much better user experience. In this lesson, we will be discussing how
-JavaScript retrieves data along with how to use the built-in `fetch()` function to
-handle remote data retrieval.
-
-## Objectives
+* Things called `Promise`s
+* Things called `XMLHttpRequestObject`s
+* A "serialization format" called JSON
```
Comment (cjbrock) [inline README.md]: I don't know about using quotation marks with these bullet points. I know that in general writing, we'd probably do this, but it's educational, and we're teaching a new vocabulary here, so if anything I think I'd put them in backticks

File: README.md [hunk 5]
```diff
@@ -1,224 +1,147 @@
...
+* "asynchronous Input / Output"
```
Comment (cjbrock) [inline README.md]: same backtick comment here

File: README.md [hunk 6]
```diff
@@ -1,224 +1,147 @@
...
+* "the event loop"
```
Comment (cjbrock) [inline README.md]: same backtick comment here

File: README.md [hunk 7]
```diff
@@ -1,224 +1,147 @@
...
 
-1. Explain how JavaScript fetches data from remote resources
-2. Explain how `fetch()` is used in modern browsers
-3. Explain what are sending `fetch()` requests 
-4. Working around backwards compatibility issues
+Understanding each of these first gets you away from _experiencing_ the magic
```
Comment (cjbrock) [inline README.md]: I'm not really understanding what "gets away from" means here. I'd rephrase a bit like this:
In order to understand the JavaScript `fetch()` technique, it's not really necessary to understand exactly how all of the about AJAX functionality works. We're going to gloss over all of these pieces in this lesson, and by doing so you'll have a solid understanding of how `fetch()` works.

File: README.md [hunk 8]
```diff
@@ -1,224 +1,147 @@
...
+> course will provide additional material to introduce you to the key
+> technology. If, on the other hand, you're not seeking that deeper technical
+> understanding, this introduction should be enough.
 
-Getting remote data in JavaScript has traditionally required a fair amount of
-plumbing to make things happen. In the past, the only option was something
-called an `XMLHttpRequest`, also referred to as XHR. It's not that it's *hard*
-to get data using XHR, but it does take quite a bit of setup.
+Let's learn to use `fetch()` to apply the AJAX technique: a way to load
+additional data _after_ information is presented to the user.
```
Comment (cjbrock) [inline README.md]: I really like this.

File: README.md [hunk 9]
```diff
@@ -1,224 +1,147 @@
...
-astronauts in space (sent back from the API we requested from), but that's a lot
-of setup to just say "give me some JSON from this URL". There are alternatives,
-such as jQuery's `$.ajax()`, that applied some syntactic sugar on top of
-`XMLHttpRequest`s, but ultimately, there should be an even simpler way to send
-requests for remote data.
+## Explain How To Fetch Data With `fetch()`
 
-## Explain How `fetch()` is Used in Modern Browsers
+The `fetch()` function retrieves data. It's a global _method_ on the `window`
+object. That means you can simply use it with `fetch()`.
```
Comment (cjbrock) [inline README.md:55]: Really nice simplification here.

File: README.md [hunk 10]
```diff
@@ -1,224 +1,147 @@
...
 ## Working Around Backwards Compatibility Issues
 
-As you can see, `fetch()` provides us with a clean, low-maintenance way to
-get and work with resources.
+As you can see, `fetch()` provides us with a short way to fetch and work with
+resources. However, `fetch()` has only recently arrived in browsers. In older
+code you might see `jquery.ajax` or `$.ajax` or an object called an
+`XMLHttpRequestObject`.
+
+## Identify Examples Of The AJAX Technique On Popular Web Sites
```
Comment (cjbrock) [inline README.md]: I like this section!

**Review by cjbrock — CHANGES_REQUESTED** [review]
Overall, nicely solid rewrite. Some style notes up top but I like the edits to remove the advanced topics and I like the instinct to remove all the AJAX content.

**Comment by sgharms** [comment]
Thanks integreated.
